examples/rust/providers/blank-slate/README.md

```diff
@@ -0,0 +1,35 @@
+# Blank Slate Capability Provider
+
+This capability provider is a blank slate for creating providers with custom capabilities. It uses the [wasmcloud-provider-sdk](https://crates.io/crates/wasmcloud-provider-sdk) and implements the [Provider](https://docs.rs/wasmcloud-provider-sdk/0.5.0/wasmcloud_provider_sdk/trait.Provider.html) trait with an example handler that will persist the links that target the provider (target links) and links where the provider is the source and targets a component (source links).
+
+## Building
+
+Prerequisites:
+
+1. [Rust toolchain](https://www.rust-lang.org/tools/install)
+1. [wash](https://wasmcloud.com/docs/installation)
+
+You can build this capability provider by running `wash build`.
+
+## Running
+
+Prerequisites:
+
+1. [Rust toolchain](https://www.rust-lang.org/tools/install)
+1. [nats-server](https://github.com/nats-io/nats-server)
+1. [nats-cli](https://github.com/nats-io/natscli)
+
+You can run this capability provider as a binary by passing a simple base64 encoded [HostData](https://docs.rs/wasmcloud-core/0.6.0/wasmcloud_core/host/struct.HostData.html) struct, in order to do basic testing. For example:
```

Comment:
This is excellent for quick testing, but what do you think about including a `wadm` file? Maybe with a commented out component that's attached to the provider?

Reply:
Love that idea!

Reply:
Also one thing I think that took me a surprisingly long time to understand was how to specify named config from a WADM file versus source/target config (and maybe a note about how you shouldn't put secrets in the file) -- it's just... in a different spot, but for some reason I couldn't figure it out for a bit.